apoc.import.csv 4.3.0.3 doesn't find URLs beginning with forward slash

## Guidelines

Please note that GitHub issues are only meant for bug reports/feature requests. If you have questions on how to use APOC, please ask on [the Neo4j Discussion Forum](https://community.neo4j.com/c/neo4j-graph-platform/procedures-apoc) instead of creating an issue here.

## Expected Behavior (Mandatory)
apoc.import.csv should find URLs starting with "file:/"

## Actual Behavior (Mandatory)
org.neo4j.driver.exceptions.ClientException: Failed to invoke procedure `apoc.import.csv`: Caused by: java.io.IOException: You're providing a directory outside the import directory defined into `dbms.directories.import`

## How to Reproduce the Problem
Example from a liquigraph script:

            CALL apoc.import.csv(
            [
                {fileName: 'file:/masterData/series/masterData_architecture.csv', labels: ['MasterData','Architecture']}
            ],
            [],
            {})


### Simple Dataset (where it's possibile)
Empty database.

### Steps (Mandatory)

  1. Run cypher with apoc.import.csv

## Specifications (Mandatory)

Currently used versions

### Versions

  - OS: Windows 10 Pro 21H1
  - Neo4j: 4.3.7-enterprise
  - Neo4j-Apoc: 4.3.0.3

I'm pretty sure this used to work in 4.3.0.3 and before, we have always used this script. With APOC 4.3.0.3, we had to remove the first slash, e.g. file:masterData/series/masterData_architecture.csv. This contradicts the documentation at https://neo4j.com/labs/apoc/4.3/overview/apoc.import/apoc.import.csv/

--- a/core/src/main/java/apoc/util/FileUtils.java
+++ b/core/src/main/java/apoc/util/FileUtils.java
@@ -6,6 +6,7 @@ import apoc.export.util.CountingReader;
 import apoc.util.hdfs.HDFSUtils;
 import apoc.util.s3.S3URLConnection;
 import apoc.util.s3.S3UploadUtils;
+import org.apache.commons.io.FilenameUtils;
 import org.apache.commons.lang3.StringUtils;
 import org.neo4j.configuration.GraphDatabaseSettings;
 
@@ -193,7 +194,7 @@ public class FileUtils {
     }
 
     private static Path relativizeIfSamePrefix(Path urlPath, Path basePath) {
-        if (urlPath.isAbsolute() && !urlPath.startsWith(basePath.toAbsolutePath())) {
+        if (FilenameUtils.getPrefixLength(urlPath.toString()) > 0 && !urlPath.startsWith(basePath.toAbsolutePath())) {
             // if the import folder is configured to be used as root folder we consider
             // it as root directory in order to reproduce the same LOAD CSV behaviour
             urlPath = urlPath.getRoot().relativize(urlPath);

